ParkSeongTeak/1546/ArrayHeader.cpp

```diff
@@ -0,0 +1,47 @@
+#include "ArrayHeader.h"
+#include<iostream>
+using namespace std;
+
+Array::Array(int size) : size(size) {
+	arr = new int [size]{ 0, };
+}
+Array::~Array() {
+	delete[] arr;
+}
+
+void Array::SetArr() {
+	for (int i = 0; i < size; i++) {
+		cin >> arr[i];

```

Comment:
SetNum() 함수와 기능이 완전 똑같은데,
```suggestion
		cin >> data;
		SetNum(i, data);
```
이런 방식으로 하시면 코드의 재사용성이 높아질 것 같습니다.

Reply:
> SetNum() 함수와 기능이 완전 똑같은데,
> 
> 이런 방식으로 하시면 코드의 재사용성이 높아질 것 같습니다.

네 저도 그렇게 생각을 하긴 합니다만 cpp 내부 구현인데 SetNum 함수를 사용하는 건 너무 비효율적이라(지역변수 int *2 + 계산= 1회) 배열 개수마다 돌아가는 특성상 기능적으로 이게 더 좋을 것 같긴 합니다